### PR TITLE
ci: Add q35 setup

### LINKED ISSUE #762: Add tests for q35 machine type
We can run the tests with q35 machine type in one of the distributions we test in CI.

### PR DESCRIPTION
**Pull request by GabyCT**
This will help us to create a CI for q35 testing.

Fixes #762

Signed-off-by: Gabriela Cervantes <gabriela.cervantes.tellez@intel.com>

### CODE REVIEW

File: .ci/install_runtime.sh [hunk 1]
```diff
@@ -103,6 +104,11 @@ else
 	echo "Kata runtime will not set as a default in Docker"
 fi
 
+if [ "$MACHINETYPE" == "q35" ]; then
```
Comment (devimc) [inline .ci/install_runtime.sh]: I thought that defining MACHINETYPE=q35 was enough

Comment (devimc) [reply .ci/install_runtime.sh]: I think `-e` option should be added to the `make` command, for example here https://github.com/kata-containers/tests/blob/master/.ci/lib.sh#L74-L103

**Comment by GabyCT** [comment]
@devimc changes applied

**Pull request merged**